src/OrbitGl/App.cpp

```diff
@@ -1673,6 +1674,19 @@ void OrbitApp::SelectFunctionsFromHashes(const ModuleData* module,
   }
 }
 
+void OrbitApp::SelectFunctionsByName(const ModuleData* module,
+                                     absl::Span<const std::string> function_names) {
+  for (const auto& function_name : function_names) {
+    const orbit_client_protos::FunctionInfo* const function_info =
+        module->FindFunctionFromPrettyName(function_name);
```

Comment:
Follow up from previous comment: we could use the hash "function_name" to retrieve the FunctionInfo